standard/docs/en/implementation/amendments.md

```diff
@@ -16,57 +16,169 @@ The nature of a change can be made explicit using:
+
+See the example release below.
+
+```eval_rst
+
+.. jsoninclude:: ../_static/json/amendments-contract-example.json
+   :jsonpointer: /records/0/releases/1
+   :expand: tag, contracts, items, amendments
+   :title: ContractAmendment
+
+```
+
+Note that amendments can cover more than values or duration. Also, note that the publisher chose to not repeat the contract items, but add a new one with a new ID value. 
+
+In certain scenarios there may not be a valid `amendsReleaseID` and so it can be omitted, e.g. when historical data is being published in a single release.
+
+#### Record
+
+An example record for the whole process is shown below. Consider downloading the <a href="../../_static/json/amendments-contract-example.json" target="_blank">record example</a> and use the [Data Review Tool](https://standard.open-contracting.org/review) to explore the changes in the contracting process.
+
+Note that the `compiledRelease` contains all the items, included the latest one added in the contract amendment.
+
+```eval_rst
+
+.. jsoninclude:: ../_static/json/amendments-contract-example.json
    :jsonpointer: 
-   :expand: records, compiledRelease, tender, tag, versionedRelease, tender, value, amount, releaseTag, amendments
-   :title: versionedRecord
+   :expand: records, releases
+   :title: Record
 
 ```
 
+### Example 3: Amendments in a Easy Releases scenario
 
-Note in this example that:
+The Easy releases guidance (<mark>add link</mark>) explains how to build releases when there is no access to the change history for contracting processes. In a scenario when little or no change history is present, it may or may not be possible to identify and build amendments inside releases.
 
-* **The publisher chooses in the tenderAmendment release to repeat a fragment of the original 'tender' release**, so that the before and after values of the tender can be accessed in the single file. This is not necessary when a full archive of releases is made accessible, but can be a convenient way of publishers making structured information on the before and after values of a field more accessible to users. 
+However, it is possible for a publisher to store partial history of events that may be relevant to them, like contract amendments. This example shows how amendment releases can be built when amendment history is present.
 
-* **In the merged record**, the information in the `amendments` array can be used to lookup information in the `versionedRelease` section, to see where changes are explained by an amendment `rationale`. 
+A publisher does not register changes in the contracting process. However, since contract amendments are a more frequent ocurrence, these are stored as new contract notices with a reference to the contract being modified. 
```

Comment:
I think this is now covered by the suggested change above, but it would be good to replace it with a link to a real life example.

[CN3350299](https://www.tenders.gov.au/Cn/Show/03a3c53e-b3bd-eac1-558a-4c659e44a516) from AusTender has many amendments.

We can add sentence to make it clear that if this was published using only a single release in OCDS, then only the latest contract value and duration would be available to users, however if each amendment is published as a separate release then the full history of changes to the value and duration is available.